src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -153,7 +170,7 @@ protected void setApiHost(String apiHost) {
     }
 
     @JsonIgnore
-    @Parameter(names = {"--remote"}, description = APIConfig.Descriptions.REMOTE)
+    @Parameter(names = {"--remote"}, description = APIConfig.Descriptions.REMOTE, arity = 1)
```

Comment:
Is not only about setting the arity. If the user passes a `false` value the functionality should be disabled.